styles/dark/markdown.mcss

```diff
@@ -1,111 +1,51 @@
 Markdown {
-  (a.bad) {
-    color: inherit
-    :hover {
-      text-decoration: inherit
-    }
-  }
-
-  word-break: break-word
-  (img) {
+  img {
```

Comment:
I've noticed that the theme files are missing _parentheses_ around the selector, but the ones in `base` are fine. I think that (due to a bug in mcss) if any instance of that rule is defined with _parentheses_, that all are treated this way. So it doesn't seem to be breaking anything in the patchwork styles. However, for consistency (and in case the mcss bug is fixed), these should be updated to have the _parentheses_ on all of these selectors.

This is handled in the tokenizer by `deep: true`. This is for choosing when not to use the direct descendent selector (`>`). Which I see is being handled by [the serializer](https://github.com/mmckegg/micro-css/pull/3/files#diff-72eec7d9bc03ee5b16aa5768cb53d614R46). Maybe there is a bug in your diffing?

Reply:
I think this might be a serializer bug where `deep: true` is being treated like `color: "red"` or something: it's preserved in the base because it's found in both token objects, but removed from the diffs because it's treated like a shared property. I think it's diffing on:

``` css
A {
  b {
    background: blue
    color: green
    deep: true
  }
}

Reply:
Turns out the `deep` issue wasn't a serializer bug, it was the diff function that wasn't passing `deep` along in the diff. I couldn't find an elegant solution I liked so I hacked together a [a solution that works](https://github.com/ssbc/patchwork/commit/c20552d7246784403f17117ed9cc775d71ff1738), which basically says:

```
if (foo.deep === true) {
  foo.deep = 1
}
```

This means that it passes our two requirements:

- it's technically different, so it's passed along to the serializer
- it's still truthy, so the serializer treats it the same as before and prints `(foo)`